src/main/java/org/openbase/bco/bcozy/view/BackgroundPane.java

```diff
@@ -61,6 +75,14 @@ public BackgroundPane(final ForegroundPane foregroundPane) throws InstantiationE
             this.editingLayerPane = new SimpleUnitSymbolsPane();
             this.editingLayerPane.setPickOnBounds(false);
 
+
+            try {
+                this.powerChartPaneAndController = FXMLProcessor.loadFxmlPaneAndControllerPair("PowerChartVisualization.fxml",  PowerChartVisualizationController.class);
```

Comment:
Please checkout if `FXMLProcessor.loadFxmlPaneAndControllerPair(PowerChartVisualizationController.class);` is working as well.

Reply:
As proven in 19fbf78 it works

Reply:
nice.